libs/dao-data/src/types/query.types.ts

```diff
@@ -122,7 +122,7 @@ export type DaoProfile = {
   longDescription?: string;
   avatarImg?: string;
   tags?: string[];
-  links?: string;
+  links?: { [field: string]: string };
```

Comment:
I still want to test this on some other daos - i feel like it might come through as a string on stuff added from our new form.